nixos/modules/config/shells-environment.nix

```diff
@@ -106,15 +106,22 @@ in
       type = types.lines;
     };
 
-    environment.shellAliases = mkOption {
-      default = {};
+    environment.shellAliases = mkOption rec {
+      default = {
+        ls = "ls --color=tty";
+        ll = "ls -l";
+        l  = "ls -alh";
+      };
       example = { ll = "ls -l"; };
```

Comment:
Maybe something other than a default could be used here, e.g. `example = { gst = "git status"; }`